src/Symfony/Component/Validator/Resources/translations/validators.gl.xlf

```diff
@@ -314,6 +314,22 @@
                 <source>This is not a valid Business Identifier Code (BIC).</source>
                 <target>Non é un Código de Identificación Bancaria (BIC) válido.</target>
             </trans-unit>
+            <trans-unit id="82">
+                <source>Error</source>
+                <target>Erro</target>
+            </trans-unit>
+            <trans-unit id="83">
+                <source>This is not a valid UUID.</source>
+                <target>Isto non é un UUID válido.</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Este valor debería ser multiplo de {{ compared_value }}.</target>
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Este Código de identificación comercial (BIC) non está asociado co IBAN {{ iban }}.</target>
```

Comment:
@Lutacon in Spanish BIC is called *Código de Identificación Bancaria* which is not a literal translation of the English term. Maybe this should be also used for Galician translation?

Reply:
It could be used. I'm not really sure because I always use BIC or SWIFT directly. I'm going to change it to _Código de identificación Bancaria_ .